.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -42,6 +42,11 @@ Briefly summarize what changes were made to accomplish the motivation above.
 This may be unnecessary for some PRs. Catch-all for detailed explanations about the implementation decisions and implications of the change.
 
 
+**Testing**
+
+Explain how you tested the changes in this pull request and how reviewers should test the changes.
```

Comment:
I think its more approachable for the committer if they just list steps for how to test